backstop/tests/toast-notification.html

```diff
@@ -30,13 +30,166 @@
     </style>
     <script src="../assets/theme.js"></script>
   </head>
-  <body class="iui-body">
+  <body
+    class="iui-body"
+    style='height: 100vh;'
+  >
     <theme-button></theme-button>
     <h1>Toast notification</h1>
     <hr />
+    <div class="iui-input-container iui-inline">
+      <div class="iui-label iui-required">Toast notifications placement</div>
+      <label
+        class="iui-radio"
+        id='top-start-option'
+      >
+        <input
+          type="radio"
+          name="radio-demo-6"
+          value="top-start"
+        />
+        <span class="iui-radio-dot">
+          <svg
```

Comment:
I wonder if it'd be better to just use native radio. The svg stuff is too verbose.

Reply:
Simplified radio buttons.